### PR TITLE
[video_player/camera] fix the return value of GstVideoPlayer::HandleGstMessage()

### PR DESCRIPTION
**Pull request by makotosato-at**
fix for #92.
If "async queue" is not handled, HandleGstMessage() should return GST_BUS_DROP.

### CODE REVIEW

File: packages/video_player/elinux/gst_video_player.cc [hunk 1]
```diff
@@ -403,6 +403,8 @@ void GstVideoPlayer::GetVideoSize(int32_t& width, int32_t& height) {
     return;
   }
 #endif  // USE_EGL_IMAGE_DMABUF
+
+  gst_caps_unref(caps);
```
Comment (HidenoriMatsubayashi) [inline packages/video_player/elinux/gst_video_player.cc:407]: Please make another pull request if the content is different from next time.

**Review by HidenoriMatsubayashi — APPROVED** [review]

File: packages/video_player/elinux/gst_video_player.cc (hunk 1 above)
Comment (makotosato-at) [reply packages/video_player/elinux/gst_video_player.cc:407]: I'm Sorry.
It was already a merged patch...
